### ISSUE TITLE
numerical input fields don't show thousands separators

### ISSUE THREAD

**Issue by adrianinsaval** [mtotschnig/MyExpenses#1242]
Probably this is not a common problem for people working with USD, EUR, etc. but when using something like PYG (1 USD = 7300 PYG) it becomes an annoyance trying to keep track of how many zeroes I've written. I constantly have to add transactions with 5 or more zeroes and it's easy to make a mistake.

If thousands separators where appropriately shown as I type it would make it much easier.

**Comment by mtotschnig** [comment]
I completely agree that displaying thousands separators would largely improve the usability of the amount input. Unfortunately this component has a lot of legacy code that makes it hard to add this feature to it, so it might not be feasible before the whole edit screen's implementation is modernized.
